CMakeLists.txt

```diff
@@ -183,6 +183,43 @@ else ()
     message(STATUS "Your C++ compiler does not support C++11. Some features may be disabled.")
 endif()
 
+
+#########################################################################
+# Determining timer with highest resolution
+include(ProfilerTimerCheck)
+set(TIMER_RESOLUTION No)
+
+# try to get best timer on windows
+if (WIN32 OR CYGWIN)
+    # on windows and cygwin try to get access QueryPerformanceCounter timer
+    TEST_TIMER (HAVE_TIMER_QUERY_PERFORMANCE_COUNTER ${CMAKE_SOURCE_DIR}/tests/timer_queryperformance.cpp) 
+
+    # if success save theese values via flow_define and flow_define_constant macros
+    if (HAVE_TIMER_QUERY_PERFORMANCE_COUNTER AND TIMER_RESOLUTION)
+        flow_define           (HAVE_TIMER_QUERY_PERFORMANCE_COUNTER)
+        # flow_define_constant  (TIMER_RESOLUTION "${TIMER_RESOLUTION}")
+    else()
+        MESSAGE (WARNING "Couldn't access QueryPerformanceCounter timer")   
+    endif()
+endif()
+
+if (NOT TIMER_RESOLUTION)
```

Comment:
So on UNIX we always use std high resolution timer? Then why the other timer tests?
It would be helpful to present observer resolution of individual timers in a comment.